ONSdigital/ras-collection-instrument
# Update packages

# Motivation and Context
ras-collection-instrument has a number of security issues.  As well as updating the vulnerable packages, a number of other packages were updated.

Structlog and cryptography were pinned for seemingly no reason, so I've unpinned them and it seems to be working fine.

# What has changed
Packages were updated and 2 files were updated to fix warnings that occurred because of this.
The docker containers that are started as part of the testing have collection instrument have had their name changed to make it less of a hassle to run.

# How to test?
Unit and acceptance tests should still pass.
NOTE:  When unit testing, you need to turn off your rabbit and postgres docker containers as these unit tests spin up ones of their own and will fail if there is already something else on that port.

# Links
https://trello.com/c/PHHH10fi/956-fix-ras-collection-instrument-security-issues-2
